### PR TITLE
Create MoM-2023-04-12

### PR DESCRIPTION
**Pull request by jordonezlucena**
Identity and Consent Management Sub-Project: minutes of the call held in April 12th

### CODE REVIEW

File: documentation/MeetingMinutes/MOM-2023-04-12.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+| Alicon | Ike Alisson |
+| Cisco | Ravi Shek | 
+| CodeB | Juss Makela |
+| DT | Dawid Wróblewski, Rafal Artych |
+| Ericsson | Jan Friman |
+| GSMA | Mark Cornall, Toyeeb Rehman, Tom van Pelt |
+| Orange | Ludovic Robert |
+| Telefónica | **Diego González (chair)**, Jesús Peña, Jose Ordonez-Lucena  |
+| T-Mobile US| Murat Karabulut |
+| Vodafone| Eric Murray, Ivan Nieto, Fraser King, Klaus Reifenrath, Sachin Kumar, Nevin Aquinas |
```
Comment (eric-murray) [inline documentation/MeetingMinutes/MOM-2023-04-12.md:18]: Alexander Boraczynski, not Fraser King

**Comment by eric-murray** [comment]
One small change, otherwise fine

File: documentation/MeetingMinutes/MOM-2023-04-12.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+The following table summarizes the main discussion that was held after document presentation.  
+
+| Company | Q/A | Comments |
+| ---- | ---| ------- | 
+| **Topic: Consent API** | |
+| GSMA | Q | Suggested the possibility to use the API from application to trigger consent capture by the operator. |
+| Telefónica | A | From TEF point of view, consent capture should be part of the authorization flow. But it is not a "closed door" for this proposal if we found the need for CAMARA.|
+| GSMA | Q | What is the scenario applicable to user consent API?|
+| Telefónica | A | The marketplace/app asks the user to provide the consent; as soon as the user provides the requested consent, the marketplace/app shall let the telco operator know that the user has provided consent for a given purpose. To that end, the marketplace/app invokes the consent API, sending in the API call the consent, so that the telco operator can store it. Our baseline proposal is to go for OIDC flow.|
+| GSMA | Q | Consent grant at run time, at setup time, or both of them?|
```
Comment (jpengar) [inline documentation/MeetingMinutes/MOM-2023-04-12.md:70]:
```suggestion
| GSMA | Q | AuthZ grant type selection at run time, at setup time, or both of them?|
```

**Review by jpengar — APPROVED** [review]
LGTM. Just one suggested change.

**Pull request merged**